client/src/tinymce-webpack.mts

```diff
@@ -1,48 +1,41 @@
-// <details>
-//     <summary>Copyright (C) 2022 Bryan A. Jones.</summary>
-//     <p>This file is part of the CodeChat Editor.</p>
-//     <p>The CodeChat Editor is free software: you can redistribute it and/or
-//         modify it under the terms of the GNU General Public License as
-//         published by the Free Software Foundation, either version 3 of the
-//         License, or (at your option) any later version.</p>
-//     <p>The CodeChat Editor is distributed in the hope that it will be useful,
-//         but WITHOUT ANY WARRANTY; without even the implied warranty of
-//         MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU
-//         General Public License for more details.</p>
-//     <p>You should have received a copy of the GNU General Public License
-//         along with the CodeChat Editor. If not, see <a
-//             href="http://www.gnu.org/licenses/">http://www.gnu.org/licenses/</a>.
-//     </p>
-// </details>
-// <h1><code>tinymce-webpack.ts</code> &mdash; imports the TinyMCE editor from
-//     NPM packages using webpack</h1>
-// <p>Import TinyMCE.</p>
+// Copyright (C) 2022 Bryan A. Jones.
```

Comment:
Does the markdown-to-html process exclude the `<summary>` and `<detail>` tags?

Reply:
Yes, it removes these tags & doesn't have a conversion for them

Reply:
OK, thanks. I'll work on this in a separate sprint.